review/tests/test_review.py

```diff
@@ -1,22 +1,56 @@
 import pytest
+import datetime
 from datetime import timedelta
 from django.utils import timezone
 from review.models import Review
 from django.template.backends import django
 
+from django.contrib.auth.models import User
+from account.models.professional import Professions, Professional
+from account.models.profile import Profile, UserType
+from account.models.client import Client
+
+CLIENT_USERNAME = 'ClientEntity'
+PROFESSIONAL_USERNAME = 'ProfessionalEntity'
+RATING = '4'
+DESCRIPTION = 'Creating a test review'
+
 
 @pytest.fixture
-def review():
-    now = timezone.now()
-    review = Review.objects.create(rating='4', description='Creating a test review...',
-                                   date_posted=now - timedelta(days=50),
-                                   client_id='client_id', professional_id='professional_id')
-    return review
+def create_review():
+    def _create_review():
+        # Client instance
+        client_user = User.objects.create_user(username=CLIENT_USERNAME, password='password',
+                                               first_name='Client', last_name='Entity', email='client@entity.com',
+                                               last_login=datetime.datetime.now(datetime.timezone.utc))
```

Comment:
Why did you make this function if it is already in the tests that Tal prepared

Reply:
Done

Reply:
> Why did you make this function if it is already in the tests that Tal prepared

Done